src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const rightRobot = {};
+  const valueArr = Object.values(robot);
+  const sortArr = valueArr.sort();
```

Comment:
you don't need array for this task, you can make loop for the object and make check for repeated keys in this loop (use hasOwnProperty for this)